test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -185,6 +185,11 @@ Describe "New-Item" -Tags "CI" {
             Pop-Location
         }
     }
+
+    It "Should display an error message when a symbolic link target is not specified" {
+        { New-Item -Name $testfile -Path $tmpDirectory -ItemType SymbolicLink } | Should -Throw -ErrorId 'ArgumentNull,Microsoft.PowerShell.Commands.NewItemCommand'
+        $ERROR | Select-Object -First 1 | Foreach-Object Exception | Foreach-Object Message | Should -Contain 'value'
```

Comment:
Please remove this. We never check "Message" because of localization.
We could check `ParamName` instead.

Also we could add a test for second condition too (for `content.ToString()`)